### PR TITLE
Update README for definitions

### PR DESCRIPTION
**Pull request by jasonwalker80**
Update README to include the definitions directory structure. adding additional structure/best practices soon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
+https://github.com/genome/analysis-workflows/wiki
 
-| Workflow | Docker Image | Main CWL |
-| --- | --- | --- |
-| unaligned_bam_to_bqsr | mgibio/dna-alignment | unaligned_bam_to_bqsr/workflow.cwl |
-| detect_variants | mgibio/cle | detect_variants/detect_variants.cwl |
-| rnaseq | mgibio/rnaseq | rnaseq/workflow.cwl |
-| tumor_only_exome | mgibio/cle | exome_workflow.cwl |
-| umi_alignment | mgibio/dna-alignment | umi_alignment/duplex_workflow.cwl |
+## Docker Images
```
Comment (jasonwalker80) [inline README.md]: @apaul7 Anything to add hear. I can include it now or we can add it after this is merged.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
 
-If a tool or workflow is a subworkflow of a larger workflow, then the same Docker image is required.
+All supported Docker images are available on mgibio DockerHub: https://hub.docker.com/u/mgibio/
 
+Many tools utilize Docker images hosted publically on DockerHub as well as [BioContainers](https://biocontainers.pro).
```
Comment (chrisamiller) [inline README.md]: I might say "hosted publicly in other repositories, such as Dockerhub or Biocontainers"

File: README.md (hunk 1 above)
Comment (apaul7) [reply README.md]: I would merge this as is and then we can add something later. We are looking into parsing the cwl tools to generate a list of docker images and the sha256 associated with them.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
 
-## Docker Images
+The structure of the repo is as follows:
+
+| Path | Description |
+| --- | --- |
+| definitions | parent directory containing all CWL tool and workflow definitions |
+| definitions/pipelines | all workflows which rely on subworkflows and tools to produce final outputs |
+| definitions/subworkflows | workflows that combine multiple tools to produce intermediate (used as inputs to other subworkflows) pipeline outputs |
+| definitions/tools | CWL that wrap command line interfaces or scripts connecting multiple tools |
```
Comment (tmooney) [inline README.md]: Should we have an entry for `defintions/types`?

File: README.md (hunk 2 above)
Comment (tmooney) [reply README.md]: A possible alternate phrasing of the whole sentence:
```
Many tools rely on third-party Docker images publicly available from sources such as Docker Hub and BioContainers.
```

**Comment by jasonwalker80** [comment]
Requested changes were made. Anything else?

**Comment by chrisamiller** [comment]
+1 merge this, as it's loads better than the current one.  We can continue to iterate and improve!

**Pull request merged**